app/app/src/main/kotlin/com/hedvig/android/app/ui/HedvigApp.kt

```diff
@@ -36,50 +35,39 @@ import androidx.compose.ui.platform.LocalDensity
 import androidx.compose.ui.unit.LayoutDirection
 import androidx.compose.ui.unit.dp
 import androidx.lifecycle.compose.collectAsStateWithLifecycle
-import androidx.navigation.NavController
 import coil.ImageLoader
 import com.hedvig.android.app.navigation.HedvigNavHost
 import com.hedvig.android.core.buildconstants.HedvigBuildConstants
 import com.hedvig.android.core.designsystem.material3.motion.MotionTokens
 import com.hedvig.android.language.LanguageService
 import com.hedvig.android.market.Market
 import com.hedvig.android.navigation.activity.ActivityNavigator
-import com.hedvig.android.navigation.core.AppDestination
 import com.hedvig.android.navigation.core.HedvigDeepLinkContainer
-import com.hedvig.android.navigation.core.TopLevelGraph
-import com.kiwi.navigationcompose.typed.navigate
 
 @Composable
 internal fun HedvigApp(
   hedvigAppState: HedvigAppState,
   hedvigDeepLinkContainer: HedvigDeepLinkContainer,
   activityNavigator: ActivityNavigator,
-  getInitialTab: () -> TopLevelGraph?,
-  clearInitialTab: () -> Unit,
   shouldShowRequestPermissionRationale: (String) -> Boolean,
   market: Market,
   imageLoader: ImageLoader,
   languageService: LanguageService,
   hedvigBuildConstants: HedvigBuildConstants,
 ) {
-  LaunchedEffect(getInitialTab, clearInitialTab, hedvigAppState) {
```

Comment:
not entirely understand what this launchedEffect did before

Reply:
When we had many more activities, a bunch of those would do something like
`LoggedInActivity.newInstance(..., initialTab = ForeverTab)`, so they'd navigate here but also inform this to also navigate to the right top level tab as soon as you've gone to LoggedInActivity.
In practice this now basically never gets called anymore, so it was time to get rid of the baggage.